agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -614,7 +615,12 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
                 ap_wlan_hal->get_radio_info().vht_center_freq;
             notification->cs_params().switch_reason = beerocks::CH_SWITCH_REASON_UNKNOWN;
             message_com::send_cmdu(slave_socket, cmdu_tx);
+            return false;
         }
+        //TODO: in case only tx power received need to send notification to slave as there
+        // is no event for this
+        if (request->tx_limit_valid())
+            ap_wlan_hal->set_tx_power_limit(request->tx_limit());
```

Comment:
We do we allow setting the tx power using a channel switch message without providing channel information?

Reply:
According to Multi-AP spec channel preference TLV could be missing, so only TX power will be received.